### ISSUE TITLE
Verify performance regression with nacos binding upgrade and add it back

### ISSUE THREAD

**Issue by shubham1172** [dapr/components-contrib#1549]
## Describe the feature
Dapr introduced the Alibaba Nacos binding with #814 in v1.2 which was then reverted back in the endgame with https://github.com/dapr/dapr/pull/3212, after a performance regression https://github.com/dapr/dapr/issues/2479.

Nacos will be upgraded to a newer version with #1478, this issue is to verify if the performance regression is still happening and add it back to daprd if it's fixed.

## Release Note

RELEASE NOTE: ADD binding for Nacos configuration service

**Comment by shubham1172** [comment]
@cinience do you want to take a look at this?

**Comment by shubham1172** [comment]
@berndverst can you please reopen this issue? It is yet to be addressed.

**Comment by yaron2** [comment]
@shubham1172 reopened.

**Comment by berndverst** [comment]
@shubham1172 was this ever resolved? The component is still not registered at this time.

**Comment by shubham1172** [comment]
> @shubham1172 was this ever resolved? The component is still not registered at this time. FYI @yaron2 @artursouza

No, the performance issue is yet to be analyzed again after the SDK was upgraded.
